### ISSUE TITLE
Needless autocomplete prompt box remains on screen

### ISSUE THREAD

**Issue by alankilborn** [notepad-plus-plus/notepad-plus-plus#12559]
### Description of the Issue
The autocompletion popup box should only be shown if it is helpful.

### Steps to Reproduce the Issue

1. Open a new Notepad++ tab
2. Type `aaa` on line 1 and press Enter
3. Type `aaa` on line 2

### Actual Behavior
![image](https://user-images.githubusercontent.com/77065706/203410285-a147e70a-9d64-4772-ad6e-2cbc8765f97c.png)

### Expected Behavior
Since I already have `aaa` on line 2, the autocomplete box is of no assistance (just a visual annoyance at this point) and should not be shown.

### Debug Information
Notepad++ v8.4.7   (64-bit)
Build time : Nov  1 2022 - 23:31:11
Path : C:\\...\npp.8.4.7\notepad++.exe
Command Line : -multiInst
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 21H2
OS Build : 19044.2130
Current ANSI codepage : 1252
Plugins : 
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4)

**Comment by iubito** [comment]
Hello,
Same annoying problem for me.

Notepad++ v8.5.6   (64-bit)
Build time : Aug 15 2023 - 15:29:28
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "C:\Program Files\Notepad++\change.log" 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit)
OS Version : 22H2
OS Build : 19045.3324
Current ANSI codepage : 1252
Plugins : 
    JsMapParser.NppPlugin (4.2)
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4)
    NppXmlTreeviewPlugin (2)
    OpenSelection (1.1.2)
    PreviewHTML (1.3.2)
    SessionMgr (1.4.3)

**Comment by victorel-petrovich** [comment]
I agree that there's no need for that box once you've finished typing the word.
